plugins/database/src/mid_level_routines.cpp

```diff
@@ -603,7 +605,7 @@ int cmlCheckNameToken( const char *nameSpace, const char *tokenName, icatSession
     int status;
 
     if ( logSQL_CML != 0 ) {
-        rodsLog( LOG_SQL, "cmlCheckNameToken SQL 1 " );
+        log_sql::debug("cmlCheckNameToken SQL 1 ");
     }
```

Comment:
Things like this can likely be removed since it provides no real value.

That includes `logSQL_CML`. Make sure you determine how the variable is set and where it's used before removing it.